src/components/Auth.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from "react";
+import { useLocation, useNavigate, Navigate } from "react-router-dom";
```

Comment:
```suggestion
import { useLocation, Navigate } from "react-router-dom";
```

The `useNavigate` hook is never used, so no need to import it.